fyle_accounting_mappings/serializers.py

```diff
@@ -265,6 +265,17 @@ def to_representation(self, data):
         return super(EmployeeMappingFilteredListSerializer, self).to_representation(data)
 
 
+class CategoryMappingFilteredListSerializer(serializers.ListSerializer):
+    """
+    Serializer to filter the active system, which is a boolen field in
+    System Model. The value argument to to_representation() method is
+    the model instance
+    """
+
+    def to_representation(self, data):
+        return super(CategoryMappingFilteredListSerializer, self).to_representation(data)
+
+
 class EmployeeMappingSerializerV2(serializers.ModelSerializer):
```

Comment:
https://fylein.slack.com/archives/C053VAAAJU9/p1695192320117999?thread_ts=1694772691.922109&cid=C053VAAAJU9 please incorporate this change too, this is for adding card account. Just 1 thing, spin up qbo app and check if this addition breaks anything